Doc/library/typing.rst

```diff
@@ -1389,11 +1389,19 @@ These are not used in annotations. They are building blocks for declaring types.
    ``Point2D.__optional_keys__``.
    To allow using this feature with older versions of Python that do not
    support :pep:`526`, ``TypedDict`` supports two additional equivalent
-   syntactic forms::
+   syntactic forms. Firstly, using a literal :class:`dict` as the
+   second argument::
 
-      Point2D = TypedDict('Point2D', x=int, y=int, label=str)
       Point2D = TypedDict('Point2D', {'x': int, 'y': int, 'label': str})
 
+   Secondly, using keyword arguments::
+
+      Point2D = TypedDict('Point2D', x=int, y=int, label=str)
+
+   .. deprecated-removed:: 3.11 3.13
+      The keyword-argument syntax is deprecated in 3.11 and will be removed
+      in 3.13. It may also be unsupported by third-party type-checking tools.
```

Comment:
```suggestion
      in 3.13. It may also be unsupported by static type checkers.
```
This is the standard term used in the documentation

Reply:
Done!